_members/2019/spring.md

```diff
@@ -0,0 +1,33 @@
+---
+semester_index: 3
+members:
+- Samuel Julson
+- Ali Zaidi
+- Tom Powers
+- Tyler Nettesheim
+- Steven Ritter
+- Stewart Hagen
+- Matthew Strimaitis
+- Tiger He
+- Matthew Kramer
+- Ethan Amundson
+- Wes Koerner
+- John "Mack" Zern
+- Graham Mendola
+- Bradley Mickelson
+- Bryce Belgarde
+- Kiara Mutschler
+- Stephen Yuan
+- Yudai Yaguchi
+- Sathwik Pattem
+- Dewansh Bista
+- Adam McAvoy
+- Oscar Zhang
+- Alexander Kofman
+- Matthew Bruton
+- Matthew Spotts
+- Tambre Hu
+- Stephanie Jasper
+- Han Cao
+- Lingzheng He
+- Vivian Shi
```

Comment:
@ajkofman, all frontmatter must be terminated with `---`. Because it is missing, Jekyll isn't processing this page. I was able to figure this out by adding `{{sorted_pages | inspect}}` to `member_index.html`. (This is the file that resolves the current semester.)